### PR TITLE
Add EXT_texture_compression_s3tc_srgb for ES

### PR DESCRIPTION
**Pull request by Groovounet**

### CODE REVIEW

**Comment by Groovounet** [comment]
I didn't know how to run the tools to produce the generated files. Is there a doc somewhere?

Thanks,
Christophe

**Comment by oddhack** [comment]
Yes, the toplevel README. You will also need to fix the conflict.

File: extensions/registry.py [hunk 1]
```diff
@@ -2314,6 +2314,12 @@
         'supporters' : { 'INTEL', 'NVIDIA' },
         'url' : 'extensions/EXT/EXT_texture_compression_s3tc.txt',
     },
+    'GL_EXT_texture_compression_s3tc_srgb' : {
+        'esnumber' : 286,
```
Comment (dgkoch) [inline extensions/registry.py]: I believe the next available ES number is 289 (please run 'python nextfree.py' in the extensions folder to find the next available number.

Comment (Groovounet) [inline extensions/registry.py]: Fixed, my clone was behind in number of extensions registred :D

**Comment by oddhack** [comment]
I've updated the gl2ext.h header with this extension now as well.

**Comment by Groovounet** [comment]
Thanks!

**Pull request merged**